db/migrate/20150504180617_create_dispatches.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDispatches < ActiveRecord::Migration
+  def change
+    create_table :dispatches do |t|
+      t.integer :driver_id
```

Comment:
on all foreign_key's you should have an [index](http://stackoverflow.com/questions/18302066/any-reason-to-not-create-indexes-on-foreign-key-columns-in-activerecord)!